tests/test_configure_feedstock.py

```diff
@@ -682,19 +701,24 @@ def test_migrator_downgrade_recipe(
         == 2
     )
 
+    conda_build_param = request.node.callspec.params["config_yaml"]
+    expected_value = "1000"
+
+    ci_support_filename = "linux_64_python2.7.yaml"
+
     with open(
         os.path.join(
             recipe_migration_cfep9_downgrade.recipe,
             ".ci_support",
-            "linux_64_python2.7.yaml",
+            ci_support_filename,
         )
     ) as fo:
         variant = yaml.safe_load(fo)
-        assert variant["zlib"] == ["1000"]
+        assert variant["zlib"] == [expected_value]
```

Comment:
Can you reduce unnecessary diff like this?

Reply:
yes, done.